spyder_kernels/console/tests/test_console_kernel.py

```diff
@@ -396,6 +396,21 @@ def test_runcell(tmpdir):
         content = msg['content']
         assert content['found']
 
+        # Verify that the `fname` variable is `cell-test.py`
+        client.inspect('fname')
+        msg = client.get_shell_msg(block=True, timeout=TIMEOUT)
+        print(msg['content'])
+        content = msg['content']
+        assert "cell-test.py" in content['data']['text/plain']
+
+        # Verify that the `__file__` variable is undefined
+        client.inspect('__file__')
+        msg = client.get_shell_msg(block=True, timeout=TIMEOUT)
+        print(msg['content'])
```

Comment:
I think the `print`s in this test are unnecessary (the first one was my fault). Could you remove them to see what happens? Thanks!

Reply:
It works fine without them. :+1: